apps/admin-gui/src/app/vos/pages/group-detail-page/group-members/group-members.component.ts

```diff
@@ -264,6 +268,18 @@ export class GroupMembersComponent implements OnInit {
     });
   }
 
+  isCopyMembersDisabled(): void {
+    this.groupService.getGroupDirectMembersCount(this.group.id).subscribe({
+      next: (count) => {
+        this.copyDisabled = count === 0;
+      },
+      error: () => {
+        this.copyDisabled = true;
+      },
+    });
+    this.copyDisabled = false;
```

Comment:
NITPICK: I would move line `this.copyDisabled = false;` to the beginning of this method before the async call due to readability. It's probably clearer to set the variable to false and wait for the async call instead of running an async call (skip waiting), set the variable to `false`, and then rewrite the variable according to the async response.